### PR TITLE
testnet manual docs

### PR DESCRIPTION
**Pull request by dzmitryhil**

### CODE REVIEW

File: deploy/testnet/scripts/init-full-node.sh [hunk 1]
```diff
@@ -56,8 +56,7 @@ mkdir -p $ONOMY_HOME
 echo '{
   "node_name": "",
   "validator_name": "",
```
Comment (Parth4337) [inline deploy/testnet/scripts/init-full-node.sh]: We can remove validator name entry

File: docs/testnet/installation.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Installation
+
+In theory, Onomy chain can be run on Windows and Mac. Binaries will be provided on the releases page and currently
+scripts files are provided to make binaries. We also suggest an open notepad or other document to keep track of the keys
+you will be generating.
+
+## Bootstrapping steps and commands
+
+Start by logging into your Linux server using ssh. The following commands are intended to be run on that machine. There
```
Comment (Parth4337) [inline docs/testnet/installation.md]: Start by logging into your Linux server ~~using ssh~~. The following commands are intended to be run on that machine...

File: docs/testnet/installation.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+To download and install binaries follow these steps
+
+1. Create a new directory in your home directory which will save all the onomy
+   packages. `mkdir -p $HOME/.onomy/bin && cd $HOME/.onomy/bin`
+2. Download binaries using wget and add executable permission
+
+```
+cd $HOME/.onomy/bin
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.3/onomyd
```
Comment (Parth4337) [inline docs/testnet/installation.md]: change v0.0.3 to latest and add the latest tag to latest stable release so we don't have to change the docs every time we release a new version

File: docs/testnet/installation.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+### <a name="rpmInstall"></a> 3. Install RPM package
+
+If you are using a Red Hat or Fedora based OS, you can use following RPM to install the binaries and download required
+script files.
+
+In order to install using RPM, use the following command:
+
+```
+sudo yum install "https://github.com/onomyprotocol/onomy/releases/download/v0.0.3/onomy-0.0.3-1.x86_64.rpm"
```
Comment (Parth4337) [inline docs/testnet/installation.md]: change v0.0.3 to latest and add latest tag to latest releases.

File: docs/testnet/full.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+Pass through the [full node](../../deploy/testnet/full.md) instruction
+
+### Check the status of the Onomy Network Testnet
+
+You should be good to go! You can check the status of the Onomy chain by running:
+
+```
+curl http://<your_ip>:26657/status
```
Comment (Parth4337) [inline docs/testnet/full.md:40]: or by visiting the same URL from the browser

File: docs/testnet/full.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+```
+ulimit -n
+```
+
+If limit is still 1024, changes are not in effect yet.
+
+### Start your full node in another terminal and wait for it to sync
+
+```
+onomyd start
```
Comment (Parth4337) [inline docs/testnet/full.md:112]: Add

You can also redirect the logs using 
```
onomyd start &>>  path/to/log/file
```

File: docs/testnet/full.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+### Check the status of the Onomy chain
+
+You should be good to go! You can check the status of the Onomy chain by running.
+
+```
+curl http://<your_ip>:26657/status
```
Comment (Parth4337) [inline docs/testnet/full.md:120]: or by visiting the same URL from the browser

File: docs/testnet/readme.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
+# Onomy Testnet
+
+At [Onomy Protocol](https://onomy.io/), we are excited to launch our first public testnet for the Onomy Network (ONET)
```
Comment (Parth4337) [inline docs/testnet/readme.md]: we are excited to launch our ~~first~~ public testnet for the Onomy Network

File: docs/testnet/readme.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+As a Cosmos Tendermint-based chain, the ONET works via a Proof of Stake consensus mechanism, where the ANOM and it's 10^18 NOM representation is bonded to
+validators either directly or through delegation. The ONET will power the Onomy Exchange (hybrid DEX) and Onomy
+Reserve (stablecoin issuance), whilst operating with 100x the efficiency of Ethereum and facilitating inter-chain asset
+trading of NOM, Onomy's stablecoins, and other crypto assets.
+
+This iteration is also focused on proving out the Gravity bridge as well as deployment methodology and other custom
+modules for the upcoming production Onomy incentivized testnet and mainnet.
+
+The documentation here will guide you through the steps needed to run full nodes, become a validator, obtain ANOM/NOM, and
+try out the Onomy Network.
```
Comment (Parth4337) [inline docs/testnet/readme.md]: The documentation here will guide you through the steps needed to run full nodes, ~~become a validator~~, obtain ANOM/NOM, and try out the Onomy Network.

File: docs/testnet/readme.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+To join us as a or discuss the current testnet, kindly get in touch with us via [Discord](https://discord.gg/27r73SYAkQ)
+. More focused discussions are possible on our [official forum](https://forum.onomy.io/).
```
Comment (Parth4337) [inline docs/testnet/readme.md]: join us as a ?? or discuss the current testnet.......

File: docs/testnet/readme.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+## Documents
+
+- [How to install Onomy](installation.md)
+- [How to Run a Full Node](full.md)
+- [How to Get ANOMs/NOMs](bonding-curve.md)
+- [How to Become a Delegator](delegation.md)
+- [How to Become a Validator](validator.md)
```
Comment (Parth4337) [inline docs/testnet/readme.md]: Remove this

Comment (dzmitryhil) [reply docs/testnet/readme.md]: We need this since it is possible but not easy :-)

File: deploy/testnet/scripts/init-full-node.sh (hunk 1 above)
Comment (dzmitryhil) [reply deploy/testnet/scripts/init-full-node.sh]: updated

File: docs/testnet/installation.md (hunk 2 above)
Comment (dzmitryhil) [reply docs/testnet/installation.md]: updated

File: docs/testnet/installation.md (hunk 3 above)
Comment (dzmitryhil) [reply docs/testnet/installation.md]: updated

File: docs/testnet/installation.md (hunk 4 above)
Comment (dzmitryhil) [reply docs/testnet/installation.md]: updated to latest/onomy-latest.x86_64.rpm

File: docs/testnet/full.md (hunk 7 above)
Comment (dzmitryhil) [reply docs/testnet/full.md:120]: updated

File: docs/testnet/readme.md (hunk 8 above)
Comment (dzmitryhil) [reply docs/testnet/readme.md]: updated

File: docs/testnet/readme.md (hunk 10 above)
Comment (dzmitryhil) [reply docs/testnet/readme.md]: removed

File: docs/testnet/readme.md (hunk 9 above)
Comment (dzmitryhil) [reply docs/testnet/readme.md]: removed

File: docs/testnet/readme.md (hunk 11 above)
Comment (dzmitryhil) [reply docs/testnet/readme.md]: We will fill the "validator.md" with the requirements for the validators

**Pull request merged**